src/arrays.js

```diff
@@ -1,49 +1,65 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) 
+  return array[index - array.length];
+  return array[index]
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString()
 };
 
-const csvStringToArray = string => {
-  // your code here
-};
+function csvStringToArray(string) {
+  return string.split(",", 5)
+}
 
-const addToArray = (element, array) => {
-  // your code here
+const addToArray = (elementX, arrayX) => {
+  arrayX.push(elementX)
 };
 
-const addToArray2 = (element, array) => {
-  // your code here
+const addToArray2 = (NEWelement, ORIGarray) => {
+  let addToArray3;
+  addToArray3 = ORIGarray.concat(NEWelement)
+  return addToArray3
```

Comment:
Not sure why the renaming of the variables here, not really necessary but if you want to its always best to stick to the same casing convention throughout, in this case camelCase:
```
const addToArray2 = (newElement, originalArray) => {
  const addToArray = originalArray.concat(newElement)
  return addToArray
```
Also, you can just declare the new array you are creating with a const and save yourself a line of code